gematria/granite/python/gnn_model_base.py

```diff
@@ -27,6 +27,7 @@
 import graph_nets
 import sonnet as snt
 import tensorflow.compat.v1 as tf
+import tf_keras as keras
```

Comment:
Do we need the module rename? This doesn't seem to be a standard rename, so for the sake clarity, I'd just use `tf_keras` in the code.

Same in the other files.

Reply:
It's not really needed, but is just what the release notes mentioning the fix did, presumably so that you would only really need to change the import line if you originally had something like `from tensorflow import keras` (which was not our case).